### PR TITLE
feat(newTextCell): add new markdown cell to menu and create keyboard shortcut

### PR DESCRIPTION
**Pull request by alexandercbooth**
Added new markdown cell feature with keybinding _Shift CmdOrCtrl M_ and added test for it.

### CODE REVIEW

File: src/main/menu.js [hunk 1]
```diff
@@ -264,6 +264,11 @@ export const edit = {
       click: createSender('menu:new-code-cell'),
     },
     {
+      label: 'New Text Cell',
+      accelerator: 'CmdOrCtrl+Shift+M',
+      click: createSender('menu:new-text-cell'),
+    },
```
Comment (rgbkrk) [inline src/main/menu.js:270]: I like that the keys for code cell and text cell are next to each other. I'm a fan of this.

File: test/renderer/menu-spec.js [hunk 2]
```diff
@@ -31,6 +31,22 @@ describe('menu', () => {
...
+      const store = dummyStore();
+      store.dispatch = sinon.spy();
+
+      menu.dispatchCreateTextCellAfter(store);
+
+      expect(store.dispatch.firstCall).to.be.calledWith({
+        type: constants.NEW_CELL_AFTER,
+        cellType: 'markdown',
+        source: '',
+        id: null,
```
Comment (rgbkrk) [inline test/renderer/menu-spec.js:45]: I thought this part was going to fail CI then I saw that the `NEW_CELL_AFTER` code uses a `null` id as well.

File: src/main/menu.js (hunk 1 above)
Comment (alexandercbooth) [reply src/main/menu.js:270]: Great, thanks!

File: test/renderer/menu-spec.js (hunk 2 above)
Comment (alexandercbooth) [reply test/renderer/menu-spec.js:45]: Hm, it looks like it did fail with AppVeyor. What should I do?

Comment (lgeiger) [reply test/renderer/menu-spec.js:45]: It looks like AppVeyor didn't start properly and I can't restart it with the UI. I try to close and reopen your PR.

Comment (alexandercbooth) [reply test/renderer/menu-spec.js:45]: Oh OK, thanks Lukas!

**Pull request merged**